README.md

```diff
@@ -1,2 +1,16 @@
 # money-to-prisoners
 Money to Prisoners public application
+
+## Getting started
+
+Requirements:
+
+- [Vagrant](https://www.vagrantup.com/) v1.5+
+- [VirtualBox](https://www.virtualbox.org) v4+ (or VMWare Fusion and
+  the Vagrant plugin)
+
```

Comment:
Possibly another heading here, as the following list are not part of the application requirements.